migrations/2023_07_08_000001_update_polls_discussion_relation_to_first_post.php

```diff
@@ -61,10 +61,6 @@
 
             $deletingPolls->delete();
         });
-
-        $schema->table('polls', function (Blueprint $table) {
-            $table->foreign('post_id')->references('id')->on('posts')->onDelete('cascade');
-        });
```

Comment:
Instead of moving this to later, why not move the stuff above it to an earlier migration. This way you won't need to do that weird DB resolving and identifying whether the constraint was already created 🙈